plugin.video.vrt.nu/resources/settings.xml

```diff
@@ -56,14 +56,16 @@
         <setting label="30863" help="30864" type="action" action="InstallAddon(plugin.video.youtube)" option="close" visible="!System.HasAddon(plugin.video.youtube)"/> <!-- Install YouTube add-on -->
         <setting label="30865" help="30866" type="bool" id="showyoutube" default="true" visible="System.HasAddon(plugin.video.youtube)"/>
         <setting label="30867" help="30868" type="action" option="close" action="Addon.OpenSettings(plugin.video.youtube)" enable="eq(-1,true)" visible="System.HasAddon(plugin.video.youtube)" subsetting="true"/> <!-- YouTube settings -->
-        <!-- UpNext -->
+        <!-- Up Next -->
         <setting label="30869" help="30870" type="action" action="InstallAddon(service.upnext)" option="close" visible="!System.HasAddon(service.upnext)"/> <!-- Install Up Next add-on -->
         <setting label="30871" help="30872" type="bool" id="useupnext" default="true" visible="System.HasAddon(service.upnext)" />
         <setting label="30873" help="30874" type="action" action="Addon.OpenSettings(service.upnext)" enable="eq(-1,true)" option="close" visible="System.HasAddon(service.upnext)" subsetting="true"/> <!-- Up Next settings -->
-        <!-- Twitter -->
-        <!-- setting label="30875" help="30876" type="action" action="InstallAddon(service.twitter)" option="close" visible="!System.HasAddon(service.twitter)"/ -->
-        <setting label="30877" help="30878" type="bool" id="usetwitter" default="true" visible="System.HasAddon(service.twitter)"/>
-        <setting label="30879" help="30880" type="action" option="close" action="Addon.OpenSettings(service.twitter)" enable="eq(-1,true)" visible="System.HasAddon(service.twitter)" subsetting="true"/>
+        <!-- IPTV Manager -->
+        <setting label="30875" help="30876" type="action" action="InstallAddon(service.iptv.manager)" option="close" visible="!System.HasAddon(service.iptv.manager)"/> <!-- Install IPTV Manager add-on -->
```

Comment:
This add-on does not yet exist right?

Reply:
It does exist and is functional, but it is not yet submitted to the Kodi repository. That is why this functionality is marked experimental.

Reply:
Indeed, so normal users will try to install it and be unsuccessful because it does not yet exists in our repo. So it should be commented here.

Reply:
@basrieter But then it couldn't be installed from the settings if it were available from elsewhere.

Reply:
So they install it via other means. The submission to the Kodi repository is for the majority of the users. And this kind of functionality should only be there if the referenced add-on exists in the Kodi repository.

Reply:
But we make one v2.3.4 release, this new review requirement (that has nowhere been documented BTW) forces us to make a different release for the Kodi repository where it is the same add-on and version.

(What we do here we do for all our integrations the same way, Youtube, Up Next, Socks)